src/main/java/com/dynatrace/loadrunner/CommandLineParser.java

```diff
@@ -0,0 +1,56 @@
+package com.dynatrace.loadrunner;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class CommandLineParser {
+
+	final public Map<String, String> arguments;
+
+	public CommandLineParser(String[] args) {
+
+		arguments = new HashMap<>();
+
+		for (int i = 0; i < args.length; i++) {
```

Comment:
check if using `Iterator` won't make it a bit easier to use and would clarify skipping some elements, as it provides method like `hasNext()`